src/models/callercalleemodel.h

```diff
@@ -263,16 +263,26 @@ class LocationCostModelImpl : public HashModel<Data::SourceLocationCostMap, Mode
     QVariant cell(int column, int role, const Data::FileLine& fileLine,
                   const Data::LocationCost& costs) const final override
     {
+        // if we have a recording with the following layout: A B C where
+        // A: part where function f is accessed
+        // B: list samples
+        // C: part where function g is accessed
+        // then f won't have the lost events cost while g does.
+        // This will cause a crash when the model tries to access it
+
+        const std::size_t costType = column - NUM_BASE_COLUMNS;
```

Comment:
const auto

Reply:
`std::valarray<>::opetator[]` takes `std::size_t` as argument so I need this to be `std::size_t` otherwise `clang-tidy` complains (auto will set the type to `int`)